main/cmds.go

```diff
@@ -61,6 +61,13 @@ func uninstall(ctx *log.Context, h vmextension.HandlerEnvironment, seqNum int) e
 }
 
 func enable(ctx *log.Context, h vmextension.HandlerEnvironment, seqNum int) error {
+	// for a few versions we need to migrate dataDirOld (introduced in v2.0.0) to
+	// dataDir (introduced in v2.0.1).
+	ctx.Log("message", "checking for state migration")
```

Comment:
The nice thing is that this is a patch version, so once 2.0.1 is everywhere we no longer have to worry about this.